tools/maptools.py

```diff
@@ -69,13 +72,17 @@ def __deploy__(self):
     def __copy__(self):
         mapcopy(self.args)
 
+    def __test__(self):
+        maptest(self.args)
+
     def run(self):
         commands = {
             'config': self.__config__, 
             'build': self.__build__,
             'package': self.__package__,
             'deploy': self.__deploy__,
-            'copy': self.__copy__
+            'copy': self.__copy__,
+            'test': self.__test__,
```

Comment:
Great idea to add a `test` command to maptools.

Note that this should be in a separate commit from the changes to prplmesh_utils.sh and the changes to .travis.yml. So in the end, there should be 3 commits I think:

1. Add `check` to prplmesh_utils.sh
2. Add `test` to maptools
3. Add call to `test` to travis.yml